### ISSUE TITLE
Include reference to client in twisted.web's docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2025]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#2025|
|Type|defect|
|Created|2006-08-29 07:01:08Z|

twisted.web is documented as "Web Server" although it contains client and server.  Expand the docstring to include a reference to client.

Attachments:

* [client-ref-2025.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/004/004be89dd9e070ecb080b9b759e5be29ec24881b/4ffcda860f93bf68e4b476f6b4b2fef2919a7348.patch) (550 bytes) - added by thijs on 2008-07-14 06:50:47Z - patch against r24269
* [web-doc.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/004/004be89dd9e070ecb080b9b759e5be29ec24881b/e8cdd388b467b0900b16731d23b17cb8c346936b.patch) (718 bytes) - added by exarkun on 2008-07-15 12:24:07Z - counter-patch

<details><summary>Searchable metadata</summary>

```
trac-id__2025 2025
type__defect defect
reporter__jml jml
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1156834868000000 1156834868000000
changetime__1216303738000000 1216303738000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Attached a [[client-ref-2025.patch](None/client-ref-2025.patch) patch] that adds a reference to the client, inc. a epydoc link.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

`client-ref-2025.patch` reads funny to me.  Alternate proposal attached.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Replying to [exarkun](#note_3):
> `client-ref-2025.patch` reads funny to me.  Alternate proposal attached.

Yep that's better, I'd say apply to trunk.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [24280]) Apply web-doc.patch

Author: exarkun
Reviewer: thijs
Fixes: [#2025](https://github.com/twisted/twisted/issues/2025)

Change the module docstring for `twisted.web` to mention the client as
well as the server.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
